tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -413,12 +448,12 @@ def test_loading_message_from_alb_fails(mocker, monkeypatch, all_path_env_param_
     mocker.patch('base64.b64decode', side_effect=base64.binascii.Error)
     response = vetext_incoming_forwarder_lambda_handler(albInvokedWithoutAddOn, False)
 
-    assert response['statusCode'] == 500
+    assert response['statusCode'] == 403
```

Comment:
This is the error code currently returned when an undecodable message is sent. From a conversation before, a ticket will be created to address this.